content/en/continuous-deployment/spinnaker-user-guides/canary/kayenta-canary-use.md

```diff
@@ -8,65 +8,57 @@ description: >
   Learn how to configure and use Kayenta for Automatic Canary Analysis in Spinnaker.
 ---
 
-{{< include "user-guide/canary/config-kayenta-frag.md" >}}
 
-## Overview of Kayenta for Automated Canary Analysis
 
-Kayenta is the Spinnaker service that performs Automated Canary Analysis (ACA). The goal of Kayenta is to provide the end user with confidence that a deployment is safe through automation and intelligence.
+## Overview of Kayenta for automated canary analysis
+
+Kayenta is the Spinnaker service that performs automated canary analysis. The goal of Kayenta is to provide the end user with confidence that a deployment is safe through automation and intelligence.
 
-Kayenta uses real-time data sources to validate that a canary is good or bad. Today, Kayenta supports the following real-time data sources:
+Kayenta uses real-time data sources to validate that a canary is good or bad. Kayenta supports the following real-time data sources:
 
 * DataDog
 * Stackdriver (Google)
 * Prometheus
 * New Relic
 * Dynatrace (Armory feature)
-* CloudWatch (Armory feature)
+* AWS CloudWatch
```

Comment:
```suggestion
* AWS CloudWatch (Armory feature)
```

We should either make sure both dynatrace and cloudwatch have `(Armory feature)`, or neither of them do.